Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -444,7 +444,7 @@ def visit_ForInStatNode(self, node):
             self.in_iterator = True
             self.visit(node.iterator)
             self.in_iterator = False
-            self.visitchildren(node, attrs=[attr for attr in node.child_attrs if attr!="iterator"])
+            self.visitchildren(node, exclude=("iterator",))
```

Comment:
I undid this change - `TreeVisitor` doesn't have `exclude` as an argument.

Reply:
Right. I moved the option up to the `TreeVisitor` class in the master branch, because why not.